DSCResources/MSFT_xIisModule/xIisModuleDesigner.ps1

```diff
@@ -30,7 +29,7 @@ $resources += New-xDscResourceProperty -Name EndPointSetup -Type Boolean -Attrib
 
 
 
-Write-Host updating...
+Write-Verbose updating...
```

Comment:
I know not strictly necessary and all, but could we wrap the updating... in quotes - just sort of feels wrong :smile: